contents/basepages/aboutus/index.en.mdx

```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
+
+### Try to change your mind
+- "A change in perspective is worth 80 IQ points." -- Alan Key (A father of personal computer.)<br/>
+  "A change of perspective is worth 80 points of IQ." -- Alan Kay, father of the personal computer
+
+So I started studying online marketing.
+I am studying business administration and software engineering at university.
+It's because I've been doing internet related work for a long time.
+
+Also, rather than trying various superficial methods, I thought that exploring fundamental and holistic methods would provide me with long-lasting knowledge.
+If you use a superficial method, when the situation changes, you have to study from scratch again...
+
+### Meet the right way for small shops
+
+Then, many small shops and companies cannot sell because
+I realized that it was because I imitated TV commercials and magazine advertisements for large companies.
+In other words, there was a method of advertising, publicity, and sales that was optimal for the size of the store and the size of the trade area.
+
+It's called DRM (Direct Response Marketing).
+Moreover, it is similar to the way of thinking and way of thinking that was done in traditional Japanese business.
+
+Do you know Drucker, who is famous for "Moshi Dora"?
+It is said that Echigoya in the Edo period started the world's oldest marketing.
+
+So, if you are Japanese, the content of my Google Maps attracting customers using DRM and DRM is
+It's all about the things that make sense honestly.
+
+However, when applying the Internet,
+Internet and computer knowledge are required.
+
+It's not easy for anyone to use.
+
+### Meet Google Maps, easy to use on your smartphone
+
+However, it is ideal for small local shops,
+Anyone can use it easily with a smartphone,
+And I found out about free Google business profiles.
+Google business profile
+It is a tool to manage the information of the shop placed on Google Maps.
+
+In fact, at that time and now, in Japan, it increases the search ranking of Google Maps
+Even if there is information on "MEO (map engine optimization) measures",
+There is not much information that actually leads to the store from there.
+
+Therefore, I decided to obtain information in English from overseas at that time.
+It turns out that the word "MEO" is unique to Japan and was not used overseas.
+
+Instead, Google officially used "Local SEO"
+(Even if there is a search engine,
+There is no such thing as a map engine. )
+
+Local SEO is not local SEO.
+It's a local search engine optimization.
+Again, this is confusing, but the English word "local" doesn't seem to mean rural or country.
+Ultimately, it is a pair of words for global (whole), and it seems that regional and local are more appropriate.
+
+Then I started researching local SEO and started attracting customers on Google Maps.
+
+
+  Because I want to spread the way to let you know
+* If you make a mistake in notifying, not only will customers not come,
+  because i lose money
```

Comment:
Please be careful of line breaks and translate them again.

Reply:
DONE